src/main/java/io/logz/logback/LogzioSender.java

```diff
@@ -69,15 +65,34 @@ public LogzioSender(String logzioToken, String logzioType, int drainTimeout, int
         queueDirectory = new File(bufferDir);
 
         if (additionalFields != null) {
-            this.additionalFields = Splitter.on(';').omitEmptyStrings().withKeyValueSeparator('=').split(additionalFields);
+
+            JsonObject logMessage = formatMessageAsJson(new Date().getTime(), "Level", "Message", "Logger", "Thread");
+
+            Splitter.on(';').omitEmptyStrings().withKeyValueSeparator('=').split(additionalFields).forEach((k, v) -> {
+
+                if (logMessage.get(k) != null) {
+                    reporter.warning("You requested to add value name " + k + ", but it is reserved. Ignoring it.");
```

Comment:
This doesn't hint me in the right direction of the source of the problem. Maybe "The field name 'k' defined in additionalFields configuration can't be used since it's a reserved field name. This field will not be added to the outgoing log messages"